tools-v2/pkg/cli/command/curvebs/update/leader_schedule_all/leader_schedule_all.go

```diff
@@ -0,0 +1,96 @@
+/*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+ */
+
+/*
+* Project: curvecli
+* Created Date: 2023-06-14
+* Author: montaguelhz
+ */
+
+package leader_schedule_all
+
+import (
+	"fmt"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+	"github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/update/leader_schedule"
+	"github.com/opencurve/curve/tools-v2/pkg/config"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/spf13/cobra"
+)
+
+type LeaderScheduleAllCommand struct {
+	basecmd.FinalCurveCmd
+}
+
+var _ basecmd.FinalCurveCmdFunc = (*LeaderScheduleAllCommand)(nil) // check interface
+
+const (
+	leaderScheduleAllExample = `$ curve bs update leader-schedule-all`
```

Comment:
use a optional flag is better?
For example: `$ curve bs update leader-schedule -all`

Reply:
I followed the issue #2352 add do-snapshot-all. if I use a flag `all`, naming styles need to be unified.  What do you think we should do?

Reply:
However, snapshot-all is not yet implemented in tools-v2(Go). We will implemented `bs snapshot copyset --all` in tools-v2 in the feature.

Reply:
@Cyber-SiKu please take a look this comment, thx~

Reply:
> However, snapshot-all is not yet implemented in tools-v2(Go). We will implement `bs snapshot copyset --all` in tools-v2 in the feature.

Cool, listen to you.